File: Problem_1/sol.js
sum5_3 = function(n) {
  var total = 0;
  for(var i=1; i < n; i ++){
    if(i % 5 === 0 || i % 3 === 0){
      total += i;
    }
  }
  console.log(total);
};
sum5_3(1000);

File: Problem_1/sol.c
#include <stdio.h>

int main() {
	int n = 1000;
	int total = 0;
	for (int i = 1; i < n; i++){
		if(i % 3 == 0 || i % 5 == 0){
			total += i;
		}
	}
	printf("%d\n", total);
	return 0;
}
